File: Zadanie 1.py
a = input("podaj pierwszą liczbę działania: ")
a = int(a)
opt = input("podaj operatora działania z podanych (+, -, *, /): ")
b = input("podaj drugą liczbę działania: ")
b = int(b)

if opt=="+":
    c = a + b
    print(f"{a} + {b} = {c}")
elif opt=="-":
    c = a - b
    print(f"{a} - {b} = {c}")
elif opt=="*":
    c = a * b
    print(f"{a} * {b} = {c}")
elif opt=="/":
    c = a / b
    print(f"{a} / {b} = {c}")

File: Zadanie 1.cs
using System;
class HelloWorld {
  static void Main() {
    Console.WriteLine("podaj pierwszą liczbę działania: ");
    float a = Convert.ToSingle(Console.ReadLine());
    Console.WriteLine("podaj operatora działania z podanych (+, -, *, /): ");
    string opt = Console.ReadLine();
    Console.WriteLine("podaj drugą liczbę działania: ");
    float b = Convert.ToSingle(Console.ReadLine());
    
    if (opt == "+")
    {
        float c = a + b;
        Console.WriteLine($"Wynik: {a} {opt} {b} = {c}");
    } else if (opt == "-")
    {
        float c = a - b;
        Console.WriteLine($"Wynik: {a} {opt} {b} = {c}");
    } else if (opt == "*")
    {
        float c = a * b;
        Console.WriteLine($"Wynik: {a} {opt} {b} = {c}");
    } else if (opt == "/")
    {
        float c = a / b;
        Console.WriteLine($"Wynik: {a} {opt} {b} = {c}");
    }
  }
}